src/envo/envo-edit.owl

```diff
@@ -39978,6 +39980,32 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/ENVO_00000015>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01001497>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105004> (thermosetting disposition)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Thermosetting_polymer>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105004> "A disposition inhering in a bearer by virtue of the bearer's ability to become permanently rigid when heated or cured."@en)
```

Comment:
>A disposition inhering in a bearer by virtue of the bearer's ability to become permanently rigid when heated or cured

Not sure that's right. Heating is part of the [curing](https://en.wikipedia.org/wiki/Curing_(chemistry)#Curing_process )process, rather than an "or". 

Perhaps (and also de-BFOising the language for broader understanding):

> A disposition which inheres in materials capable of becoming rigid when cured by heating.